admin-ui/plugins/auth-server/redux/api/AgamaApi.js

```diff
@@ -14,15 +14,15 @@ export default class AgamaApi {
     })
   }
 
-  addAgama = ({payload}) => {
-    const {file, name, token} = payload
+  addAgama = ({ payload }) => {
+    const { file, name, token } = payload
     return new Promise((resolve, reject) => {
-        axios.post('/api/v1/agama-deployment/' + name, file, {
-          headers: {
-            Authorization: 'Bearer ' + token,
-            'Content-Type': 'application/zip',
-          }
-        })
+      axios.post('/api/v1/agama-deployment/' + name, file, {
```

Comment:
we should use jans-config-api client for calling apis.

Reply:
@duttarnab Yes, we could, but I encountered an error with the jans-config-api because we are using a string. That's why we are opting for Axios instead.